LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-computed-contrast-color-function.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Test CSS Color Module 5 contrast-color() computed value</title>
+<link rel="author" title="Sam Weinig" href="mailto:sam@webkit.org">
+<link rel="help" href="https://drafts.csswg.org/css-color-5/#contrast-color">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/computed-testcommon.js"></script>
+<style>
+  #container {
+    container-type: inline-size;
+    width: 10px;
+    color: pink;
+  }
+</style>
+</head>
+<body>
+<div id="container">
+  <div id="target"></div>
+</div>
+<script>
+
+// NOTE: As the algorithm for determining constrast is not defined, all computed values can validly be either `rgb(0, 0, 0)` or `rgb(255, 255, 255)`.
+
+// Test basic legacy colors
+test_computed_value("background-color", "contrast-color(white)", ["rgb(0, 0, 0)", "rgb(255, 255, 255)"]);
+test_computed_value("background-color", "contrast-color(black)", ["rgb(0, 0, 0)", "rgb(255, 255, 255)"]);
```

Comment:
There's probably some obvious ones like `contrast-color(white)` / `contrast-color(black)` where we can assume are `black` / `white`. Given this requirement:

> However, colors returned by this function should still meet the WCAG 2.1 [section 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum) for AA large text, as many authors need to meet legal requirements that mandate this.